.github/workflows/build.yml

```
@@ -144,6 +146,19 @@ jobs:
         CMAKE_ARGS="-DAPPLE_CODESIGN_IDENTITY=${APPLE_CODESIGN_IDENTITY}"
         echo "::set-output name=cmake_args::${CMAKE_ARGS}"
 
+    - name: "[Windows] Set up Windows code signing"
+      if: runner.os == 'Windows' && env.WINDOWS_CODESIGN_CERTIFICATE_PASSWORD != null && env.WINDOWS_CODESIGN_CERTIFICATE_PATH != null && env.WINDOWS_CODESIGN_CERTIFICATE_SECURE_FILE_SALT != null && env.WINDOWS_CODESIGN_CERTIFICATE_SECURE_FILE_SECRET != null
```

This is getting skipped. I don't know why. But there is no point checking WINDOWS_CODESIGN_CERTIFICATE_PATH
```suggestion
      if: runner.os == 'Windows' && env.WINDOWS_CODESIGN_CERTIFICATE_PASSWORD != null && env.WINDOWS_CODESIGN_CERTIFICATE_SECURE_FILE_SALT != null && env.WINDOWS_CODESIGN_CERTIFICATE_SECURE_FILE_SECRET != null
```

---

This is still getting skipped.

---

You could try `secrets.WINDOWS_CODESIGN_CERTIFICATE_SECURE_FILE_SALT != null`